### DISCUSSION TITLE
Enable MA as an entity so a lovelace media card can control playback

### DISCUSSION THREAD

**Discussion by starman2k01** [music-assistant/support#3140] [Feature requests and ideas]
Once a speaker is chosen, yes you can control media play from MA. But I have a couple dozen speakers and want to control MA at any time playing through any speaker. Like youtube music player allows.
I'd like a MA card for lovelace.  Your UI is great but not the normal way to control media (that is via lovelace).
Am I missing something obvious?

**Comment by OzGav** [comment]
To be honest I’m not following what you are meaning with this request but how is it not a duplicate of https://github.com/orgs/music-assistant/discussions/954
